utils/analytics.js

```diff
@@ -5,30 +5,36 @@ export const initGA = () => {
     [
       {
         trackingId: process.env.GA_UA,
-        gaOptions: { name: "tracker_1" }
+        gaOptions: { name: "tracker_1", siteSpeedSampleRate: 100 } // This will send 100% of hits to Google Analytics. By default only 1% are sent.
       },
       {
-        trackingId: process.env.GA_UA_CDS
+        trackingId: process.env.GA_UA_CDS,
+        gaOptions: { name: "tracker_2", siteSpeedSampleRate: 100 }
       }
     ]
-    // , { debug: true }
+    //, { debug: true }
   );
-  ReactGA.set({ anonymizeIp: true });
+  ReactGA.set({ anonymizeIp: true }, ["tracker_1"]);
+  ReactGA.set({ anonymizeIp: true }, ["tracker_2"]);
 };
 
 export const logPageView = () => {
-  ReactGA.set({ page: window.location.pathname });
-  ReactGA.pageview(window.location.pathname);
+  ReactGA.set({ page: window.location.pathname }, ["tracker_1"]);
```

Comment:
```
 ga("tracker_1.set", "page", window.location.pathname);
 ga("tracker_1.send", "pageview", window.location.pathname);
 ga("tracker_2.set", "page", window.location.pathname);
 ga("tracker_2.send", "pageview", window.location.pathname);
```
Also works, but fails the tests because ga isn't declared in them.